putData.sh

```diff
@@ -95,6 +95,25 @@ LOCAL_SQL_PATH=./$FROM_WIKI_DOMAIN/db.sql
 # TODO run all jobs
 
 # Fill QueryService
+WBS_DOMAIN=wikibasederp.wbaas.localhost
+QS_POD=queryservice-55fcdbcf86-x7db2
+QS_NAMESPACE=qsns_b247111900
+
+## To clear a namespace from things
+## curl 'http://localhost:9999/bigdata/namespace/qsns_b247111900/sparql' -X POST --data-raw 'update=DROP ALL;'
+
+CLOUD_KUBECTL="kubectl --context=minikube-wbaas"
+MW_POD=$($CLOUD_KUBECTL get pods -l app.kubernetes.io/name=mediawiki,app.kubernetes.io/component=app-backend -o jsonpath="{.items[0].metadata.name}")
+kubectl exec -it "$MW_POD" -- bash -c "WBS_DOMAIN=$WBS_DOMAIN php w/extensions/Wikibase/repo/maintenance/dumpRdf.php --output /tmp/output.ttl"
+
+## TODO Copy between pods instead of to local disk
+kubectl cp "$MW_POD":/tmp/output.ttl /tmp/output.ttl
```

Comment:
For the real version of this change we would want to delete the .ttl from disk once it is copied

Reply:
Done!